cetmix_tower_server/wizards/cx_tower_command_run_wizard_view.xml

```diff
@@ -80,6 +80,7 @@
                     </div>
                     <field
                         name="path"
+                        widget="ace_tower"
```

Comment:
This works, however need to add the same widget to the "code" field too.

<img width="640" height="266" alt="Image" src="https://github.com/user-attachments/assets/06a2be09-3ed0-4a3d-9fa2-054c1b3f0956" />